src/api/Controllers/CoordinateController.cs

```diff
@@ -64,33 +45,17 @@ public async Task<IActionResult> MigrateAsync(bool onlyMissing = true, bool dryR
         {
             foreach (var borehole in context.Boreholes)
             {
-                var transformDirection = borehole.OriginalReferenceSystem == ReferenceSystem.LV95 ? Lv95ToLv03 : Lv03ToLv95;
-                var sourceLocationX = transformDirection == Lv95ToLv03 ? borehole.LocationX : borehole.LocationXLV03;
-                var sourceLocationY = transformDirection == Lv95ToLv03 ? borehole.LocationY : borehole.LocationYLV03;
-                var destinationLocationX = transformDirection == Lv95ToLv03 ? borehole.LocationXLV03 : borehole.LocationX;
-                var destinationLocationY = transformDirection == Lv95ToLv03 ? borehole.LocationYLV03 : borehole.LocationY;
-                var setDestinationLocationX = (double x) => transformDirection == Lv95ToLv03 ? borehole.LocationXLV03 = x : borehole.LocationX = x;
-                var setDestinationLocationY = (double y) => transformDirection == Lv95ToLv03 ? borehole.LocationYLV03 = y : borehole.LocationY = y;
-
-                if (onlyMissing && destinationLocationX != null && destinationLocationY != null) continue;
-                if (sourceLocationX == null || sourceLocationY == null) continue;
-
-                var reframeOptions = FormattableString.Invariant(
-                    $"{transformDirection}?easting={sourceLocationX}&northing={sourceLocationY}&format=json");
-
-                using var response = await httpClient.GetAsync(new Uri(reframeOptions, UriKind.Relative)).ConfigureAwait(false);
-                response.EnsureSuccessStatusCode();
+                // Select the coordinates before migration for later comparisation.
```

Comment:
Q: Wo wird `coordinatesBeforeMigration` verwendet?

REQ: Typo fixen
```suggestion
                // Select the coordinates before migration for later comparison.
```

Reply:
War überflüssig.